apis/nucleus/src/components/listbox/components/useTempKeyboard.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+
+export function removeInnnerTabStops(container) {
+  container?.querySelectorAll('[tabIndex="0"]').forEach((elm) => {
+    elm.setAttribute('tabIndex', -1);
+  });
+}
+
+export function getVizCell(container) {
+  return container?.closest('.njs-cell') || container?.closest('.qv-gridcell');
+}
+
+// Emulate the keyboard hook, until we support it in the Listbox.
+export default function useTempKeyboard({ containerRef, enabled }) {
+  const [keyboardActive, setKeyboardActive] = useState(false);
+
+  const innerTabStops = !enabled || keyboardActive;
+
+  const keyboard = {
+    enabled,
+    active: keyboardActive,
+    innerTabStops, // does keyboard permit inner tab stops
+    outerTabStops: !innerTabStops, // does keyboard permit outer tab stops
+    blur: (resetFocus) => {
```

Comment:
maybe add
```
if (!enabled) {
    return;
}
```
I am a bit unsure how we want to keyboard navigation to work when keyboard.enabled is false.

Reply:
Not sure either, but if I add that it breaks the current implementation. 🤷